core/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -143,51 +159,43 @@ public void testFillsAttributes() throws IOException {
     }
 
     public void testWithIndexAnalyzers() throws IOException {
-
         AnalyzeRequest request = new AnalyzeRequest();
-        request.analyzer("standard");
         request.text("the quick brown fox");
+
         request.analyzer("custom_analyzer");
-        request.text("the qu1ck brown fox");
         AnalyzeResponse analyze = TransportAnalyzeAction.analyze(request, AllFieldMapper.NAME, null, indexAnalyzers, registry, environment);
         List<AnalyzeResponse.AnalyzeToken> tokens = analyze.getTokens();
-        assertEquals(4, tokens.size());
+        assertEquals(3, tokens.size());
+        assertEquals("quick", tokens.get(0).getTerm());
+        assertEquals("brown", tokens.get(1).getTerm());
+        assertEquals("fox", tokens.get(2).getTerm());
 
-        request.analyzer("whitespace");
-        request.text("the qu1ck brown fox-dog");
+        request.analyzer("standard");
         analyze = TransportAnalyzeAction.analyze(request, AllFieldMapper.NAME, null, indexAnalyzers, registry, environment);
         tokens = analyze.getTokens();
         assertEquals(4, tokens.size());
-
-        request.analyzer("custom_analyzer");
-        request.text("the qu1ck brown fox-dog");
-        analyze = TransportAnalyzeAction.analyze(request, AllFieldMapper.NAME, null, indexAnalyzers, registry, environment);
-        tokens = analyze.getTokens();
-        assertEquals(5, tokens.size());
+        assertEquals("the", tokens.get(0).getTerm());
+        assertEquals("quick", tokens.get(1).getTerm());
+        assertEquals("brown", tokens.get(2).getTerm());
+        assertEquals("fox", tokens.get(3).getTerm());
 
         request.analyzer(null);
-        request.tokenizer("whitespace");
-        request.addTokenFilter("lowercase");
-        request.addTokenFilter("wordDelimiter");
-        request.text("the qu1ck brown fox-dog");
+        request.tokenizer("standard");
         analyze = TransportAnalyzeAction.analyze(request, AllFieldMapper.NAME, null, indexAnalyzers, registry, environment);
         tokens = analyze.getTokens();
-        assertEquals(5, tokens.size());
+        assertEquals(4, tokens.size());
         assertEquals("the", tokens.get(0).getTerm());
-        assertEquals("qu1ck", tokens.get(1).getTerm());
+        assertEquals("quick", tokens.get(1).getTerm());
```

Comment:
This one tests using a tokenizer instead of a fully built analyzer. I'll push a comment explaining